rm src/elorott.sh src/elorott.desktop && git reset --hard
patch -Np1 -i "${mypatd:-"../"}/elorott.patch"
test ! -f configure && autoreconf -vif && ./configure --prefix=/usr
test -f src/elorott.svg || cp ../unibuild/data/rottexpr.svg src/elorott.svg
chmod 755 src/elorott.sh
diff a/src/elorott.sh b/src/elorott.sh
--- a/src/elorott.sh	1969-12-31 17:00:00.000000000 -0700
+++ b/src/elorott.sh	1969-12-31 17:00:00.000000000 -0700
@@ -0,0 +1,51 @@
+#!/bin/bash
+gmdir="$HOME/games/rott"
+gmcfg="$HOME/.rott"
+gmtgt="darkwar.wad"
+gmprm="window resolution 800x600"
+glnch="elorott"
+
+gconf () {
+test -d "$gmcfg" || mkdir -p "$gmcfg"
+echo -e "gamepath= $gmdir\ngamemod= $gmtgt\ngameparm= $gmprm" | tee "$gmcfg/basedir"
+}
+
+gstart () {
+echo "cd $gmbdr && $glnch $gmpar" | xargs -i -r sh -c "{}"
+}
+
+gtest () {
+gmbse="$(grep -Pi '^gamepath=.*' "$gmcfg/basedir" | head -1)"
+gmgme="$(grep -Pi '^gamemod=.*' "$gmcfg/basedir" | head -1)"
+gmgpm="$(grep -Pi '^gameparm=.*' "$gmcfg/basedir" | head -1)"
+gmbdr="${gmbse#gamepath= }"
+gmgnm="${gmgme#gamemod= }"
+gmpar="${gmgpm#gameparm= }"
+gmgnm="${gmgnm,,}"
+}
+
+test -f "$gmcfg/basedir" || gconf
+gtest
+if [ -z "$gmbdr" ]; then
+  gconf
+  gtest
+fi
+if [ -d "$gmbdr" ]; then
+  if [ ! -f "$gmbdr/$gmgnm" ]; then
+    gmchk="$(find "$gmbdr" -mindepth 1 -maxdepth 1 -type f -regextype posix-extended -iregex "$gmbdr/$gmgnm" -printf '%p\n' | grep -Pic "$gmbdr/$gmgnm")"
+    if [ $gmchk -eq 1 ]; then
+      echo -e "File system is case sensitive\nFile/folder names must be lowercase\n Rename subdirs of\n  $gmbdr\n    containing\n  $gmtgt?\nRequires rename command" | xmessage -file - -buttons Yes,No -default No -center -timeout 30
+      test $? -eq 101 && find "$gmbdr" -depth -exec rename 's|([^/]*\Z)|lc($1)|e' {} +
+    elif [ $gmchk -gt 1 ]; then
+      echo -e "File system is case sensitive\nFile/folder names must be lowercase\n Cannot rename subdirs of\n  $gmbdr\n    containing\n  $gmtgt\n\nBecause more than one exists" | xmessage -file - -buttons Ok:0 -center -timeout 30
+    else
+      echo -e "Config:\n $gmcfg/basedir\n  $gmbse\n found, but not\n  $gmgme\n ($gmbdr/$gmgnm)\n\nRebuild config?" | xmessage -file - -buttons Yes,No -default No -center -timeout 30
+      test $? -eq 101 && gconf
+    fi
+  else
+    gstart
+  fi
+else
+  echo -e "Config:\n $gmcfg/basedir\n  $gmbse\n not found\n\nRebuild config?" | xmessage -file - -buttons Yes,No -default No -center -timeout 30
+  test $? -eq 101 && gconf
+fi
diff a/src/elorott.desktop b/src/elorott.desktop
index e69de29..feb733a 100644
--- a/src/elorott.desktop	1969-12-31 17:00:00.000000000 -0700
+++ b/src/elorott.desktop	1969-12-31 17:00:00.000000000 -0700
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=elorott
+Comment=Rise of the Triad Engine.
+Keywords=game;engine;
+Exec=/usr/share/elorott/elorott.sh
+Icon=elorott
+Terminal=false
+Type=Application
+Categories=GNOME;GTK;Game;
+StartupNotify=false
diff a/configure.ac b/configure.ac
index 074100b..5a28b6e 100644
--- a/configure.ac	1969-12-31 17:00:00.000000000 -0700
+++ b/configure.ac	1969-12-31 17:00:00.000000000 -0700
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.69])
-AC_INIT([rott], [1.1.2])
+AC_INIT([elorott], [1.1.2])
 AM_INIT_AUTOMAKE([foreign no-define])
 AC_CONFIG_SRCDIR([src/version.h])
 #AC_CONFIG_HEADERS([config.h])
@@ -65,23 +65,5 @@ esac
 SUFFIX=${suffix}
 AC_SUBST([SUFFIX])
 
-# Set data file directory
-AC_ARG_ENABLE([datadir],
-              [AS_HELP_STRING([--enable-datadir],
-                              [set data file directory])],
-              [datadir="$enableval"],
-              [datadir=no])
-
-# Remove duplicate and trailing slashes
-datadir=`echo ${datadir} | tr -s '/' | sed 's|/*$||'`
-
-case "$datadir" in
-no|yes)
-    ;;
-*)
-    AC_DEFINE_UNQUOTED([DATADIR], ["$datadir/"])
-    ;;
-esac
-
 AC_CONFIG_FILES([Makefile src/Makefile src/audiolib/Makefile])
 AC_OUTPUT
diff a/src/Makefile.am b/src/Makefile.am
index 2481d51..f398045 100644
--- a/src/Makefile.am	1969-12-31 17:00:00.000000000 -0700
+++ b/src/Makefile.am	1969-12-31 17:00:00.000000000 -0700
@@ -1,6 +1,13 @@
 SUBDIRS = audiolib
 
-bin_PROGRAMS = rott@SUFFIX@
+dist_pkgdata_SCRIPTS = elorott.sh
+Applicationsdir = $(datadir)/applications
+Applications_DATA = elorott.desktop
+pixmapdir = $(datadir)/icons/hicolor/scalable/apps
+pixmap_DATA = elorott.svg
+EXTRA_DIST = elorott.svg elorott.desktop
+
+noinst_PROGRAMS = rott@SUFFIX@
 rott@SUFFIX@_SOURCES = \
 	byteorder.c \
 	cin_actr.c \
@@ -51,3 +58,7 @@ rott@SUFFIX@_SOURCES = \
 	z_zone.c
 rott@SUFFIX@_CFLAGS = @SDL2_CFLAGS@ @SDL2_mixer_CFLAGS@ @libADLMIDI_CFLAGS@
 rott@SUFFIX@_LDADD = audiolib/libaudiolib.a @SDL2_LIBS@ @SDL2_mixer_LIBS@ @libADLMIDI_LIBS@
+
+install-data-hook:
+	cp rott $(DESTDIR)$(pkgdatadir)/elorott
+	ln -rsf $(DESTDIR)$(pkgdatadir)/elorott $(DESTDIR)${exec_prefix}/games/
